l1-contracts/contracts/bridge/L1ERC20Bridge.sol

```diff
@@ -64,10 +64,9 @@ contract L1ERC20Bridge is IL1ERC20Bridge, ReentrancyGuard {
     function initialize() external reentrancyGuardInitializer {}
 
     /// @dev transfer token to shared bridge as part of upgrade
-    function transferTokenToSharedBridge(address _token, uint256 _amount) external {
+    function transferTokenToSharedBridge(address _token) external {
```

Comment:
You should change the function in IL1ERC20Bridge.sol, because it doesn't compile